resources/public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,1981 @@
+                      arrival:
+                        href: /customs/transits/movements/arrivals/1
+                    arrivalId: '1'
+                    messageId: '1'
+                    received: 2020-12-12T12:12:15
+                    messageType: IE007
+                    body: <CC007A>...</CC007A>
+                  arrival:
+                    id: '1'
+                    created: 2020-10-10T10:10:10
+                    updated: 2020-12-12T12:12:12
+                    movementReferenceNumber: MRN
+                    _links:
+                      self:
+                        href: /customs/transits/movements/arrivals/1
+                      messages:
+                        href: /customs/transits/movements/arrivals/1/messages
+        '406':
+          description: 'Not Acceptable'
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/errorResponse'
+                - example:
+                    code: ACCEPT_HEADER_INVALID
+              examples:
+                badRequest:
+                  description: The accept header is missing or invalid.
+                  value:
+                    code: ACCEPT_HEADER_INVALID
+      deprecated: false
+      security:
+        - userRestricted:
+            - common-transit-convention-traders
+
+  /customs/transits/movements/arrivals/{arrivalId}/messages/{messageId}:
+    get:
+      tags:
+      - common-transit-convention-traders
+      summary: Get a cached message related to an arrival and message ID
+      description: >
+        Retrieve all messages related to a specific arrival and message ID that have been received within the past 31 days.
+        
+        For example, this could include messages about route diversions or unloading permissions.
+        
+        Movements older than 31 days and their related messages are archived and cannot be returned by the API.
+
+        **Note:** You can view a transit movement only if your Economic Operators Registration and Identification (EORI) number matches the EORI number that created the transit declaration for that movement. Your EORI number is part of the bearer token that you use for API authentication.
+      operationId: GetACachedMessageRelatedToAnArrivalAndMessageID
+      parameters:
+        - $ref: '#/components/parameters/acceptHeader'
+        - in: path
+          name: arrivalId
+          description: The arrival ID specifying the arrival to return.
+          required: true
+          style: simple
+          schema:
+            type: string
+            example: '1'
+        - in: path
+          name: messageId
+          description: The message ID specifying the message to return.
+          required: true
+          style: simple
+          schema:
+            type: string
+            example: '1'
+      responses:
+        '200':
+          description: 'OK'

```

Comment:
There's no need to put response descriptions in quote marks.

See also previous comment about pattern that should be used for all 200 response descriptions.